src/runtime/cli/config/configuration-acrn.toml.in

```diff
@@ -150,6 +150,10 @@ block_device_driver = "@DEFBLOCKSTORAGEDRIVER_ACRN@"
 
 #debug_console_enabled = true
 
+# Agent connection dialing timeout value in seconds
+# (default: 30)
+# dial_timeout = 30
```

Comment:
I noticed that in the configuration file, config items have no leading spaces after `#`, so you may need to fix like this:
```diff
- # dial_timeout = 30
+ #dial_timeout = 30
```